abs_to_wkt/main.ipynb

```diff
@@ -0,0 +1,150 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "36236f82-9786-4fee-bc9b-e6d944241be3",
+   "metadata": {},
+   "source": [
+    "## Absolute Pixel to WKT\n",
+    "\n",
+    "This notebook demonstrates how to convert geometries stored in “absolute” format( x/y coordinates in pixel values) to WKT format (0->100) before uploading to the portal.\n",
+    "\n",
+    "This example uses:\n",
+    "- `shapely` library for geometric operations on polygon objects."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "033ac9e3-6dcb-43bd-9431-032bb959414e",
+   "metadata": {},
+   "source": [
+    "### Input Image and Pixel Coordinates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "id": "7df70153-489b-4a66-8909-e154b445b8cf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from shapely.geometry import Polygon\n",
+    "\n",
+    "# Input your values here:\n",
+    "image_file_path = \"example/input/example_image.jpg\"  # Replace with the actual path to the image file\n",
+    "json_file_path = \"example/input/example_input.json\"  # Replace with the actual path to the JSON file with the pi\n",
+    "\n",
+    "# Extract pixel dimensions of the provided image\n",
+    "image = Image.open(image_file_path)\n",
+    "image_width, image_height = image.size\n",
+    "\n",
+    "# Load image points from the JSON file\n",
+    "image_points = load_points_from_json(json_file_path)\n"
```

Comment:
I'm not sure if `load_points_from_json` is referring to something I'm not seeing. If so, I would add it. Or you can swap for:
```python
with open(json_file_path, 'r') as json_file:
    image_points = json.load(json_file)
```

Reply:
ah sorry i accidentally deleted that function while cleaning up!